manifest.json

```diff
@@ -33,7 +32,8 @@
   "content_scripts": [
     {
       "matches": ["https://*/*", "http://*/*"],
-      "css": ["src/browser_action/viewer.css"]
+      "css": ["src/browser_action/viewer.css"],
+      "js": ["src/browser_action/lodash-debounce-custom.js"]
```

Comment:
hm weird I would have expected `vitals.js` to be in the same inclusion as lodash, but I'm not super familiar with how this extension is bundled.

Reply:
I think this stems from vitals.js being injected using `executeScript` (see below) vs. lodash being made available to the content scripts this way. I'll take an AI to look at whether vitals.js should also be loaded this way for consistency (IIRC there were issues loading it as a content script).

https://github.com/GoogleChrome/web-vitals-extension/blob/a932e9973cb1587611cfe3e24ee44b1e662ad05c/src/bg/background.js#L44